src/bls.mjs

```diff
@@ -1452,32 +1108,25 @@ async function gatherSigningFinishImpl( optsSignOperation ) {
         return;
     }
     if( ! optsSignOperation.bHaveResultReportCalled ) {
-        const strErrorMessage = cc.error( "Failed BLS sign result awaiting(2): " ) +
-            cc.warning( "No reports were arrived" ) +
-            + "\n";
-        if( log.verboseGet() >= log.verboseReversed().error ) {
-            if( log.id != optsSignOperation.details.id )
-                log.write( strErrorMessage );
-            optsSignOperation.details.write( strErrorMessage );
+        if( log.id != optsSignOperation.details.id ) {
+            log.error( "Failed BLS sign result awaiting(2): {err}",
+                "No reports were arrived" );
         }
+        optsSignOperation.details.error( "Failed BLS sign result awaiting(2): {err}",
```

Comment:
Do we need error here?

Reply:
yes, it's usually timeout error when more than `5` nodes are off-line

Reply:
it's visible in development-colored version onlu